src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheck.java

```diff
@@ -120,7 +120,7 @@ public class UnusedLocalVariableCheck extends AbstractCheck {
     /**
      * An array of blocks in which local anon inner classes can exist.
      */
-    private static final int[] CONTAINERS_FOR_ANON_INNERS = {
+    private static final int[] TYPEDECL_BODY_DECLARATION_TOKENS = {
```

Comment:
```suggestion
    private static final int[] ANONYMOUS_CLASS_PARENT_TOKENS = {
```
Please do this, we can avoid extension of spelling list, and this is more obvious in the context of a check.